git-coverage

```diff
@@ -811,6 +835,12 @@ def main(argv):
                 print_patch_hunks(patch, to_print, coverage, output)
                 printed_any = 1
 
+            if all([line.new in coverage
```

Comment:
If a file has several patches, this is going to print multiple times for that file.
This shouldn't be done on a per-patch basis.